packages/playground/src/components/Playground.tsx

```diff
@@ -389,8 +183,9 @@ export default function Playground({ themes, validators }: PlaygroundProps) {
               theme={theme}
             >
               <FormComponent
-                {...templateProps}
                 {...liveSettings}
+                ArrayFieldTemplate={ArrayFieldTemplate}
+                ObjectFieldTemplate={ObjectFieldTemplate}
```

Comment:
Dang, I totally missed this, twice. Once while I was doing v5 upgrades and again just a bit earlier. The `ArrayFieldTemplate` and `ObjectFieldTemplate` are no longer direct props on `Form` instead they need to be passed via `templates`
```suggestion
                templates={{ ArrayFieldTemplate, ObjectFieldTemplate }}
```